DigitalLearningSolutions.Data.Tests/Helpers/SectionContentTestHelper.cs

```diff
@@ -15,11 +15,6 @@ public SectionContentTestHelper(SqlConnection connection)
             this.connection = connection;
         }
 
-        public IEnumerable<OldTutorial> TutorialsFromOldStoredProcedure(int progressId, int sectionId)
```

Comment:
Are there other helper methods like this which we can remove, as I think we removed most if not all of the tests comparing our results with the old stored procedures now? 🙂